### PR TITLE
[Hive-138762] - Styleguide creation

### PR DESCRIPTION
**Pull request by randypasos**
#### Changes proposed in this pull request:
- All basic styles, components, and, structure to start building the website.

|Status|Statement|Value|
|----|----|----|
|:heavy_check_mark:|Code was validated with the project linter|-|
|:heavy_check_mark:|No broken or unnecessary images|-|
|:heavy_check_mark:|Development tested on Chrome|-|
|:x:|Development tested on Firefox|-|
|:x:|Development tested on Safari|-|
|:x:|Development tested on IE11|-|
|:heavy_check_mark:|Responsive changes|-|
|:x:|CMS database changes|-|
|:x:|API Changes|-|
|:x:|Backend Changes|-|
|:heavy_check_mark:|Frontend Changes (Styles, JS, markup)|-|
|:x:|Copy Changes|-|
|:x:|Link Changes|-|
|:x:|301 Redirects|-|
|:x:|SEO Metatags|-|
|:x:|Media updates|-|
|:x:|README file was updated with the relevant information of the project|-|

GITHUB: :heavy_check_mark: Accepted :x: Rejected :warning: Pending

Screenshot 2023-04-14 at 13 26 59
Screenshot 2023-04-14 at 13 27 17
Screenshot 2023-04-14 at 13 27 34

### CODE REVIEW

File: WebSite/WebSite/Styles/components/_button.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
+﻿.#{$prefix}-c-btn {
+    border-radius: 35px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_button.scss:2]: use rem function

File: WebSite/WebSite/Styles/components/_checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿.#{$prefix}-c-checkbox {
+    &.form-check-input {
+        background-color: $color-white;
+        border-color: $color-black;
+        border-radius: 0;
+        border-width: 3px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_checkbox.scss:6]: use rem function

File: WebSite/WebSite/Styles/components/_checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        width: 30px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_checkbox.scss:7]: use rem function

File: WebSite/WebSite/Styles/components/_checkbox.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        height: 30px;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_checkbox.scss:8]: use rem function

File: WebSite/WebSite/Styles/components/_checkbox.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        box-shadow: none;
+
+        &:checked {
+            background-image: url('../img/icons/icon_checked.png');
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_checkbox.scss:12]: dont' use  absolute paths /img

File: WebSite/WebSite/Styles/components/_search-bar.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+            box-shadow: none;
+        }
+
+        &::placeholder {
+            color: $color-tuna;
+        }
+    }
+    
+    .input-group-text {
+        background: $color-las-palmas url('../img/icons/icon_magnifying-glass.svg') center no-repeat padding-box;
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/components/_search-bar.scss:26]: don't use absolute paths

File: WebSite/WebSite/Styles/core/_utils.scss [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+.#{$prefix}-m-section {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: rem(500);
+  position: relative;
+  width: 100%;
+
+  &--top {
+    background-image: url('../img/home/home-hero.png');
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/core/_utils.scss:17]: don't use absolute paths

File: WebSite/WebSite/Styles/core/_utils.scss [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  &--middle {
+    background-color: gray;
+    margin-top: -#{math.div($skewed-top, 2) - $skew-adjustment};
+    padding-top: math.div($skewed-top, 2) - $skew-adjustment;
+  }
+
+  &--bottom {
+    background-image: url('../img/home/Graphic_Circles_Grey.svg');
```
Comment (aldortsa) [inline WebSite/WebSite/Styles/core/_utils.scss:27]: don't use absolut paths

**Review by aldortsa — CHANGES_REQUESTED** [review]

File: WebSite/WebSite/Styles/components/_button.scss [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+        display: inline-flex;
+
+        &::before {
+            content: '';
+            display: block;
+        }
+
+        &--arrow {
+            &::before {
+                background: transparent url('../img/icons/icon_angle_left_solid.svg') 0% 0% no-repeat padding-box;
```
[the remark this answers is not in the document]
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_button.scss:55]: Done.

File: WebSite/WebSite/Styles/components/_button.scss [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+            }
+        }
+        
+        &--download {
+            &::before {
+                background: transparent url('../img/icons/icon_file_PDF_dark.svg') 0% 0% no-repeat padding-box;
```
[the remark this answers is not in the document]
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_button.scss:61]: Done

File: WebSite/WebSite/Styles/components/_button.scss (hunk 1 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_button.scss:2]: Done.

File: WebSite/WebSite/Styles/components/_checkbox.scss (hunk 2 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_checkbox.scss:6]: Done.

File: WebSite/WebSite/Styles/components/_checkbox.scss (hunk 3 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_checkbox.scss:7]: Done.

File: WebSite/WebSite/Styles/components/_checkbox.scss (hunk 4 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_checkbox.scss:8]: Done.

File: WebSite/WebSite/Styles/components/_checkbox.scss (hunk 5 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_checkbox.scss:12]: Done.

File: WebSite/WebSite/Styles/components/_search-bar.scss (hunk 6 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/components/_search-bar.scss:26]: Done.

File: WebSite/WebSite/Styles/core/_utils.scss (hunk 7 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/core/_utils.scss:17]: Done.

File: WebSite/WebSite/Styles/core/_utils.scss (hunk 8 above)
Comment (randypasos) [reply WebSite/WebSite/Styles/core/_utils.scss:27]: Done.

**Pull request merged**